packages/react-native-worklets/Common/cpp/worklets/SharedItems/Shareables.cpp

```diff
@@ -153,6 +153,18 @@ jsi::Value makeShareableInitializer(
   return ShareableJSRef::newHostObject(rt, shareable);
 }
 
+jsi::Value makeShareableFunction(
+    jsi::Runtime &rt,
+    jsi::Function function ) {
+  std::shared_ptr<Shareable> shareable;
+  if (function.isHostFunction(rt)) {
```

Comment:
I wanted to split this into two separate functions, but I don't know how I can check if the function is a host function on the JS side ? I did a little research but couldn't find the answer, @tjzel @tomekzaw do you have any idea ?